src/llm/apis/openai_completions.cpp

```diff
@@ -128,6 +131,45 @@ ov::Tensor load_image_stbi(const std::string& imageBytes) {
         SharedImageAllocator{image, desiredChannels, y, x});
 }
 
+static size_t appendChunkCallback(void* downloadedChunk, size_t size, size_t nmemb,
+    void* image) {
+    size_t realsize = size * nmemb;
+    auto& mem = *static_cast<std::string*>(image);
+    mem.append(static_cast<char*>(downloadedChunk), realsize);
+    return realsize;
+}
+#define CURL_SETOPT(setopt) \
```

Comment:
What do you think? Adding `_func_call` could be the indicator what we are passing here
```suggestion
#define CURL_SETOPT(setopt_func_call) \
```